Tools/Scripts/generate-gpup-webgl

```diff
@@ -470,9 +489,9 @@ class cpp_func(object):
         return True
 
     def is_implemented(self):
-        #" in a.type.type_name for a in self.args.args):
+        # " in a.type.type_name for a in self.args.args):
         #    return False
-        #if "GCGLsync" in self.return_type.type_name:
+        # if "GCGLsync" in self.return_type.type_name:
```

Comment:
Is there any benefit to keeping the commented out special case for `GCGLsync` around?